packages/closer/pages/roles/index.tsx

```
@@ -0,0 +1,204 @@
+                      {roles.map((role) => (
+                        <div
+                          key={role._id}
+                          className="border border-gray-200 rounded-lg p-6 hover:shadow-md transition-shadow"
+                        >
+                          <div className="flex flex-col sm:flex-row justify-between gap-4">
+                            <div className="flex-1">
+                              <h3 className="text-xl font-semibold mb-2">
+                                {role.title}
+                              </h3>
+                              <p className="text-gray-600 mb-4">
+                                <span dangerouslySetInnerHTML={{ __html: role.description || '' }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)